src/integrations/prefect-dbt/prefect_dbt/cli/commands.py

```diff
@@ -53,8 +53,14 @@ async def trigger_dbt_cli_command(
         dbt_cli_profile: Profiles class containing the profile written to profiles.yml.
             Note! This is optional and will raise an error if profiles.yml already exists
             under profile_dir and overwrite_profiles is set to False.
-        **shell_run_command_kwargs: Additional keyword arguments to pass to
-            [shell_run_command](https://prefecthq.github.io/prefect-shell/commands/#prefect_shell.commands.shell_run_command).
+        create_artifact: If True, creates a Prefect artifact on the task run
+            with the dbt build results using the specified artifact key.
+            Defaults to True.
+        artifact_key: The key under which to store
+            the dbt build results artifact in Prefect.
+            Defaults to 'dbt-seed-task-summary'.
+        extra_command_args: Additional command arguments to pass to the dbt command.
+            These arguments get appended to the command that gets passed to the dbtRunner client.
```

Comment:
Can you add an example showing usage of this new parameter?

Reply:
Added an example to this as well as changed the example below on line 120